src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -111,4 +112,37 @@ class Wallet(
   fun removeAccountSigner(sourceAddress: String, signerAddress: String): Transaction {
     return addAccountSigner(sourceAddress, signerAddress, 0)
   }
+
+  // Submit transaction
+  fun submitTransaction(signedTransaction: Transaction, serverInstance: Server = server): Boolean {
+    val maxRetries = 3
```

Comment:
Retries are sometimes necessary evil. We should probably let the caller decide how to handle retries.

If it is necessary to implement retries, we should parameterize the retries count and cool-down period.

Reply:
:+1: I'll remove retries for now. We can decide later how we'd like to handle it. @JakeUrban